shoop/admin/modules/products/views/forms.py

```diff
@@ -6,9 +6,17 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from __future__ import unicode_literals
+
 from django import forms
+from django.core.exceptions import ValidationError
+from django.forms import BaseModelFormSet
+from django.forms.formsets import DEFAULT_MIN_NUM
+from django.forms.formsets import DEFAULT_MAX_NUM
+from django.utils.translation import ugettext_lazy as _
 
-from shoop.core.models import Product, ShopProduct
+from filer.models import Image
+from shoop.admin.forms.widgets import MediaChoiceWidget
+from shoop.core.models import Product, ShopProduct, ProductMedia, ProductMediaKind, Shop
 from shoop.core.models.attributes import AttributeType, Attribute
```

Comment:
Consolidate this import line onto the previous one. (The models are available via `.models` as well as in the submodules.)